### ISSUE TITLE
Installing it on ubuntu server?

### ISSUE THREAD

**Issue by NinjaTurtle007** [rauchg/slackin#358]
I just want to install it on our ubuntu server. 
Not sure how and if I can to install "now" there, but isn't there straight forward way to just have slackin running?

**Comment by lauzis** [comment]
Agreed that "how to" steps could be added for ubuntu env in read me.
But steps are something like this.

1. download project
2. go to project dir
3. npm install --unsafe-perm
4. then run command ./bin/slackin --coc "$SLACK_COC" --channels "$SLACK_CHANNELS" --port $PORT $SLACK_SUBDOMAIN $SLACK_API_TOKEN $GOOGLE_CAPTCHA_SECRET $GOOGLE_CAPTCHA_SITEKEY

atleast so far i got, still testing... will write if success

**Comment by yourhead** [comment]
@lauzis two quick questions:

what type of token did you request from slack? new or legacy?
if it's a new token, what "permissions" did you request for the token. the list is crazy long and nothing looks appropriate. thanks.
